From 62c2c000645613cb6be82f48fc641c07ca25370c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20Hor=C3=A1k?= <dan@danny.cz>
Date: Tue, 29 Mar 2011 10:50:08 +0200
Subject: [PATCH 68/70] cmsfs-fuse: Enlarge fsname string

Description: cmsfs-fuse: Enlarge fsname string
Symptom:     Truncated device part of file system name
Problem:     The device part of the file system name was limited to 50
             characters and gets truncated for long device names, for
             instance if /dev/disk/by-path/ device names are used.
Solution:    Increase the fsname string limit to 200 characters.
---
 cmsfs-fuse/cmsfs-fuse.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cmsfs-fuse/cmsfs-fuse.c b/cmsfs-fuse/cmsfs-fuse.c
index 9f1aa1a..a3a16d0 100644
--- a/cmsfs-fuse/cmsfs-fuse.c
+++ b/cmsfs-fuse/cmsfs-fuse.c
@@ -46,7 +46,7 @@ struct list text_type_list;
 FILE *logfile;
 
 #define PAGE_SIZE	0xfff
-#define FSNAME_MAX_LEN	50
+#define FSNAME_MAX_LEN	200
 #define MAX_FNAME	18
 
 #define CMSFS_OPT(t, p, v) { t, offsetof(struct cmsfs, p), v }
-- 
1.7.4

